src/component/handlers/composite-android/DraftEditorCompositionHandlerAndroid.js

```diff
@@ -80,9 +128,11 @@ const deriveSelectionFromDOM = (editor: DraftEditor): SelectionState => {
 };
 
 var DraftEditorCompositionHandlerAndroid = {
-  update: (editor, editorState) => {
-    editor.setMode('edit');
-    editor.update(EditorState.set(editorState, { inCompositionMode: false }));
+  commit: (editor, editorState) => {
+    const selection = deriveSelectionFromDOM(editor);
+    const nextEditorState = EditorState.acceptSelection(editorState, selection);
+    editor.update(EditorState.set(nextEditorState, { inCompositionMode: false }));
```

Comment:
Readability suggestion: It might be worth breaking this out into a function like `const setCompositionMode = (editorState, inCompositionMode ) => EditorState.set(editorState, { inCompositionMode });`

Reply:
Seems reasonable, I've extracted the helper method.
I could probably stand to do more of that in general, thanks!